example/gluon/wavenet/README.md

```diff
@@ -0,0 +1,53 @@
+# WaveNet with Gluon
+
+Gluon implementation of [WaveNet: A Generative Model for Raw Audio](https://arxiv.org/abs/1609.03499)
+
+![net_structure1](assets/net_struc1.png)
+![net_structure2](assets/net_struc2.png)
+
+## Requirements
+- Python 3.6.1
+- Mxnet 1.2
+- tqdm
+- scipy.io
+
+
+## Usage
+
+- arguments
+  - batch_size : Define batch size (defualt=64)
+  - epochs : Define the total epoches (default=1000)
+  - mu : Define mu value for [mu-law algorithm](https://en.wikipedia.org/wiki/%CE%9C-law_algorithm) (default=128)
+  - n_residue : Define number of residue (default=24)
+  - dilation_depth : Define dilation depth (default=10)
+  - use_gpu : whether or not to use the GPU (default=True)
+  - generation : whether or not to generate a wave file for model (default=True)
+
+###### default setting
+```
+python main.py
+``` 
+or
+
+###### manual setting
+```
+python main.py --batch_size=32 --epoches=100 ...
+```
+## Train progress
+###### 0 epoch
+![epoch0](assets/progress_epoch0.png)
```

Comment:
Please store all the images in https://github.com/dmlc/web-data/tree/master/mxnet/example

Reply:
I am trying to store all images in https://github.com/dmlc/web-data/tree/master/mxnet/example.

Reply:
complete changing image link